src/zeek-setup.cc

```diff
@@ -1083,7 +1083,15 @@ SetupResult setup(int argc, char** argv, Options* zopts)
 			}
 
 		if ( options.ignore_checksums )
+			{
+			const auto& id = global_scope()->Find("ignore_checksums");
```

Comment:
True, yeah ... but zeek-setup.cc has a mix of both, and for cases where an ID needs to be retrieved because it is about to have a value set (as opposed to merely grabbing a value from it), `global_scope()->Find()` is used throughout. It's all minor and we can definitely make a pass over the whole file to straighten it out. I'll leave this as is for now.

Reply:
I hadn't looked for second-order consistencies :-) Sounds good.